src/Stryker.Core/Stryker.Core/Reporters/IReporter.cs

```diff
@@ -13,6 +14,6 @@ public interface IReporter
         // Will get called when a mutation has been tested
         void OnMutantTested(IReadOnlyMutant result);
         // Will get called when all mutations have been tested
-        void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent);
+        void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent, TestProjectsInfo testProjectsInfo);
```

Comment:
So I don't know if you still accept design suggestions, but just saying that here you're extending an interface with something that's only used in one (of many) implementations. Probably not such a big deal but you could rather have this as a dependency for the JsonReporter only and adapt the code accordingly.

Reply:
Alright, makes sense.